molgenis-navigator/src/main/frontend/src/store/actions.js

```diff
@@ -66,7 +80,14 @@ function getPackageQuery (query: string) {
  * @param query
  */
 function getEntityTypeQuery (query: string) {
-  return '/api/v2/sys_md_EntityType?sort=label&num=1000&q=(label=q="' + encodeURIComponent(query) + '",description=q="' + encodeURIComponent(query) + '");isAbstract==false'
+  const rsql = transformToRSQL({
+    operator: 'OR',
+    operands: [
+      {selector: 'label', comparison: '=q=', arguments: query},
+      {selector: 'description', comparison: '=q=', arguments: query}
+    ]
+  })
+  return ENTITY_TYPE_ENDPOINT + '?sort=label&num=1000&q=' + encodeRsqlValue(rsql) + ';isAbstract==false'
```

Comment:
The last bit is actually an `{ operator: 'AND', operands: [<<the current rsql tree>>, {selector: 'isAbstract', comparison: '==', arguments: 'false'}]}`